### ISSUE TITLE
TonelWriter must write methods "as-is" without using the source filter

### ISSUE THREAD

**Issue by eMaringolo** [instantiations/tonel-vast#140]
TonelWriter relies on the #sourceString method that does some code formatting, which can change the originally compiled code.

We should use an equivalent version of #sourceString without passing the resulting source from the source filter.

**Comment by eMaringolo** [comment]
Adriaan Van Os suggested using the following instead of #sourceCode.

```smalltalk
CompiledMethod>>#sourceStringTonel
    "Return the source code for the receiver."

    "Bypassing CompiledMethod>>#sourceString that manipulates the source.
     https://vast-support.instantiations.com/support/tickets/80256
     https://github.com/instantiations/tonel-vast/issues/140"

    ^EmLibrary cache methodSourceAt: self filePointer decompress: true
```

I'd only change the selector to `tonelSourceString`.

**Comment by eMaringolo** [comment]
Closed as per 9e84fbf commit.
